components/scream/scripts/const-maker

```diff
@@ -42,7 +42,10 @@ def const_maker(varname, expr):
     # them double-precision. The assignment to real(rtype) will truncate if
     # necessary.
     print("Fortran:")
-    print("real(rtype), public, parameter :: {} = {:22.15d} ! {}".format(varname, val, expr))
+    s = "real(rtype), public, parameter :: {} = {:22.15e} ! {}".format(varname, val, expr)
+    s = s.replace('e+','d+')
```

Comment:
why was this necessary?

Reply:
The "d" in ":22.15d" is correct for integers but not floats and thus was a bug in const-maker. @ambrad provided me with a patch to fix this bug and asked me to include it as part of this PR.